### PR TITLE
Port to vcpkg

### PR DESCRIPTION
**Pull request by remz1337**
I ported this library to vcpkg package manager for easy installation in other projects. For this, I added the install target in CMake, and fixed the relative paths of header files across the src folder.

Also updated the README for installation instruction with VCPKG.

I am assuming my other PR (#23 ) gets merged too. If not, I can drop those commits from this PR. Let me know

### CODE REVIEW

**Review by mateuszpawlik — CHANGES_REQUESTED** [review]
All is fine. I'd only like some documentation links to using VCPKG.

File: src/command_line/main.cc [hunk 1]
```diff
@@ -36,56 +44,72 @@ int main(int argc, char** argv) {
...
+  //rusage before_rusage;
+  //rusage after_rusage;
+  //timeval runtime_utime;
+  //timeval runtime_stime;
+  //unsigned int runtime;
 
   // Verify parameters.
-  if (argc != 5) {
-    std::cerr << "Incorrect number of parameters." << std::endl;
+  if (argc != 6) {
```
[the remark this answers is not in the document]
Comment (remz1337) [reply src/command_line/main.cc:56]: I copied the `main.cc` from the other branch so there shouldn't be any conflict when accepting both PRs

File: README.md [hunk 2]
```diff
@@ -27,6 +27,20 @@ Execute ``make test`` to run all tests (currently there are only correctness tes
...
 In the ``build`` directory you find the binary ``ted`` that executes the algorithms from command line. **Currently there is no help or documentation for this binary.**
 
+### Usage
+
+Example execution:
+``./ted string {x{a}} {x{b}} lgm 1``
+
+## Installing the library with VCPKG
+
+This library has been ported to VCPKG package manager. To include in your project, install with ``./vcpkg install tree-similarity``, then include the library in your project's CMakeFile
```
[the remark this answers is not in the document]
Comment (remz1337) [reply README.md:37]: Added a link to the Github repo, which provides all the necessary information to install it and get started

**Comment by remz1337** [comment]
Changes done!

**Comment by mateuszpawlik** [comment]
@remz1337, please resolve the conflicts. Merging #23 must have caused it.

**Comment by remz1337** [comment]
Should be good now

**Comment by mateuszpawlik** [comment]
Thank you @remz1337. I'm merging in.

**Comment by remz1337** [comment]
Thank you @mateuszpawlik. Can you add a tag/release of the current version? It will be easier to identify/follow future versions with vcpkg.
My open PR with vcpkg to include tree-similarity in their database:
https://github.com/microsoft/vcpkg/pull/30901

**Comment by mateuszpawlik** [comment]
With @thuetter we scheduled a little hackathon, mainly for the documentation. Is it easy to update that tag on vcpkg side?

**Comment by remz1337** [comment]
pretty easy yes, just need to update the commit hash and version string in the portfile (json file defining the library in vcpkg). I would advise that you create a tag with the current HEAD (76e62627438ea603c9881221f7444c7cb5407b6d) that matches the version string in VCPKG (but it's not mandatory).

Any further revision/release of the library will also need to be updated in vcpkg (otherwise people installing from vcpkg will always download `v0.1`)

**Pull request merged**